### PR TITLE
Add Mint CLI commands to command palette

### PR DESCRIPTION
**Pull request by lukepighetti**
This adds the following tasks as commands and resolves https://github.com/s0kil/mint-vscode/issues/9

-`build`
-`compile`
-`docs`
-`format`
-`init` (interactive)
-`install`
-`loc`
-`start`
-`test`
-`version`

recommend squash & merge

RFC: I noticed you're using `pnpm` instead of `npm`. Is that a hard requirement to contribute?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,18 +11,28 @@
...
+    "onCommand:mint.version"
   ],
   "main": "./out/src/extension",
   "devDependencies": {
     "@types/node": "^13.13.4",
-    "@types/vscode": "^1.44.0",
     "js-yaml": "^3.13.1",
-    "typescript": "^3.8.3"
+    "typescript": "^3.8.3",
+    "vscode": "^1.1.37"
```
Comment (s0kil) [inline package.json:31]: Let's remove the `vscode` dependency, it brings in security vulnerabilities.

Reference: https://code.visualstudio.com/updates/v1_36#_splitting-vscode-package-into-typesvscode-and-vscodetest

Comment (s0kil) [reply package.json:31]: Here is how it worked earlier:
https://github.com/s0kil/mint-vscode/commit/7089de5ffd6936ad7b421bfeae5c4df08f9f78b4

Comment (lukepighetti) [reply package.json]: Thanks for the explanation, I have resolved it

File: package.json [hunk 2]
```diff
@@ -28,11 +28,11 @@
     "@types/node": "^13.13.4",
     "js-yaml": "^3.13.1",
     "typescript": "^3.8.3",
-    "vscode": "^1.1.37"
+    "@types/vscode": "^1.44.0"
   },
   "scripts": {
     "convert": "js-yaml syntaxes/mint.tmLanguage.yaml > syntaxes/mint.tmLanguage.json",
-    "postinstall": "node ./node_modules/vscode/bin/install && tsc"
```
Comment (lukepighetti) [inline package.json]: I got this line from https://github.com/jrieken/vscode-formatter-sample/blob/master/package.json#L37, but without the `vscode` dep, `node ./node_modules/vscode/bin/install` doesn't work. I'm not sure what it was for. Do we still need this?

Comment (s0kil) [reply package.json]: Do not need it, we can simply use typescript `tsc` to build the extension.

Comment (s0kil) [reply package.json]: This is also how Flutter does it, https://github.com/Dart-Code/Flutter/blob/master/package.json

**Pull request merged**